<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lesson 02</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>261</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Задание 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Задание 2</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>241</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ex2_print_button">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>190</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Показать список</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ex2_add_button">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>40</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ex2_remove_button">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>80</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ex2_view_mode">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>130</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Режим иконок</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Задание 3*</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>401</width>
      <height>151</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <column>
     <property name="text">
      <string>No</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Имя компьютера</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>IP адрес</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>MAC адрес</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>box</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>192.168.1.12</string>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string>a0:78:17:82:5a:8d</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>acacia</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>192.168.1.22</string>
     </property>
    </item>
    <item row="2" column="3">
     <property name="text">
      <string>36:f2:39:bd:33:44</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>peyote</string>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string>192.168.1.15</string>
     </property>
    </item>
    <item row="3" column="3">
     <property name="text">
      <string>d2:a4:22:2e:f5:87</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="ex3_color_button">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>310</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Покрасить выделенные строки</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ex3_clear_color_button">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>360</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Снять окраску на всей таблице</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>35</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&quot;руб.&quot; или &quot;#@RUB@&quot; заменяется на &quot;₽&quot;</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>52</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&quot;евро&quot; или &quot;#@EUR@&quot; заменяется на &quot;€&quot;</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>69</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&quot;#@PPM@&quot; заменяется на &quot;‰&quot;</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>87</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&quot;#@COP@&quot; заменяется на &quot;©&quot;</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>105</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&quot;#@REG@&quot; заменяется на &quot;®&quot;</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>20</width>
      <height>241</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>260</y>
      <width>821</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
